pkg/virt-controller/services/multus_annotations.go

```diff
@@ -75,6 +89,22 @@ func GenerateMultusCNIAnnotationFromNameScheme(namespace string, interfaces []v1
 	return "", nil
 }
 
+func newBindingPluginMultusAnnotationData(kvConfig *v1.KubeVirtConfiguration, pluginName, namespace, podIfaceName string) (*networkv1.NetworkSelectionElement, error) {
+	plugin := netbinding.ReadNetBindingPluginConfiguration(kvConfig, pluginName)
+	if plugin == nil {
+		return nil, fmt.Errorf("couldnt find network binding plugin '%s' in Kubevirt configuration", pluginName)
+	}
+
+	netAttachDefNamespace, netAttachDefName := getNamespaceAndNetworkName(namespace, plugin.NetworkAttachmentDefinition)
+	return &networkv1.NetworkSelectionElement{
+		Namespace: netAttachDefNamespace,
+		Name:      netAttachDefName,
+		CNIArgs: &map[string]interface{}{
+			"name": podIfaceName,
+		},
```

Comment:
This needs to be documented on the commit and PRm it defines an API between kubevirt and the custom binding CNI.

And in this context, I do not think `name` is expressing what is provided here. ("name" of what?)

Reply:
> This needs to be documented on the commit and PRm it defines an API between kubevirt and the custom binding CNI.

Done 

> And in this context, I do not think name is expressing what is provided here. ("name" of what?)

Changed to 'vm-network-name' following offline discussion.